packages/sdk/src/services/realtime-api.ts

```diff
@@ -179,7 +179,7 @@ export class RealtimeAPI implements BaseRealtime {
 
     const unsubscribe = listenToDocumentChange(
       {
-        path: this.dataPath,
+        path: `${this.dataPath}/channels/${this.channel}`,
```

Comment:
it may be `this.dataPath`, for V3 channels `dataPath` already points to `chatRoom/{chatRoomId}/channels/{channelId}`